05/app.js

```diff
@@ -1,9 +1,13 @@
 const stats = {
   paragraphs: {
     p1: 0,
+    p2: 0,
+    p3: 0,
   },
   links: {
     '/dolor.html': 0,
+    '/adipisicing-elite.html': 0,
+    '/consectetur.html': 0,
```

Comment:
Dodawanie w ten sposób, nie rozwiązuje problemu. Co jeśli pojawią się nowe paragrafy w treści strony? Nowe linki? Będzie trzeba dodać kolejne pozycje - to będzie bardzo męczące ;)